### ISSUE TITLE
Migrate identify API requests from Things to Authentication service

### ISSUE THREAD

**Issue by blokovi** [absmach/magistrala#1386]
**ENHANCEMENT**
1. Describe the enhancement you are requesting. Enhancements include:
   - tests
   - code refactor
   - documentation
   
Currently, there is separate HTTP authentication listener on **Things** service dedicated for identify endpoints.
This listener should be shut down and leave only one HTTP listener on Things service. Endpoints openned on Things-Auth (on port `MF_THINGS_AUTH_HTTP_PORT`) should be migrated to **Auth** service.

This migration should be done after #1347 issue is resolved.

2. Indicate the importance of this enhancement to you (must-have, should-have, nice-to-have).
Should-have

3. Note
These endpoints were openned with #1385 pull request, it can be helpful to take a look at it.
